llvm/lib/Target/LoongArch/LoongArchFloat32InstrInfo.td

```diff
@@ -22,6 +25,7 @@ def SDT_LoongArchFTINT : SDTypeProfile<1, 1, [SDTCisFP<0>, SDTCisFP<1>]>;
 def SDT_LoongArchFRECIPE : SDTypeProfile<1, 1, [SDTCisFP<0>, SDTCisFP<1>]>;
 def SDT_LoongArchFRSQRTE : SDTypeProfile<1, 1, [SDTCisFP<0>, SDTCisFP<1>]>;
 
+def loongarch_brcond : SDNode<"LoongArchISD::BRCOND", SDTBrcond, [SDNPHasChain]>;
```

Comment:
Why need to define a new node? Is it same as `brcond`?

Reply:
In this patch, `ISD::BRCOND` is custom-lowered: for integer comparisons, it is transformed into `LoongArchISD::BR_CC` before instruction selection; for floating-point comparisons, it is transformed into `LoongArchISD::BRCOND` to avoid falling into a recursive lowering loop.

Reply:
I see. It's better to add these comments in code.